src/openms/source/FILTERING/DATAREDUCTION/FeatureFindingMetabo.cpp

```diff
@@ -329,7 +304,10 @@ namespace OpenMS
 
   FeatureFindingMetabo::~FeatureFindingMetabo()
   {
-
+    if(isotope_filt_svm_ != nullptr)
```

Comment:
coudl isotope_filt_svm_ be a unique_ptr and svm_free_and_destroy_model(&isotope_filt_svm_); be called in the lambda passed for destruction?  then this destructor could be default

Reply:
yes please

Reply:
has to be done in all of SVMWrapper though.

Reply:
This should be addressed in a general SVM cleanup